### PR TITLE
registry api implementation

### PR DESCRIPTION
**Pull request by kkoralsky**
this is for #723. 

Im willing to add documentation too

### CODE REVIEW

**Comment by gpocentek** [comment]
Hi @kkoralsky

Thank you for this patch! Would you mind rebasing your change? That will help re-running the CI tests too.

Thanks!

**Comment by kkoralsky** [comment]
@gpocentek i've rebased the PR and added some docs. this should be done, although i encourge to merge #760 as well to make: `gitlab project-registry-tag delete_in_bulk --project-id x --repository-id y` functional

File: docs/gl_objects/repositories.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  + :attr:`gitlab.v4.objects.Project.registries`
+
+* Gitlab API: https://docs.gitlab.com/ce/api/container_registry.html
+
+Examples
+--------
+
+Get the list of container registry repositories associated with the project::
+
+      registries = project.registries.list()
```
[the remark this answers is not in the document]
Comment (kkoralsky) [reply docs/gl_objects/repositories.rst]: ah, yes this should be repository/repositories. I've replaced that

**Comment by max-wittig** [comment]
Thanks for your contribution!

**Pull request merged**